microsoft-edge/privacy-whitepaper/index.md

```diff
@@ -747,20 +747,38 @@ To enable diagnostic data collection, go to `edge://settings/privacy`. Turn on t
 <!-- ====================================================================== -->
 ## Writing assistance
 
-Microsoft Edge provides enhanced spelling and grammar checking as you type in the browser.  To use enhanced spelling and grammar checking, go to `edge://settings/languages` and select **Microsoft Editor** under the **Use writing assistance** toggle. 
+To help you write faster and with fewer mistakes on the web, Microsoft Edge provides writing assistance tools, including enhanced spelling and grammar checking and text prediction. 
 
-If you select **Microsoft Editor**, Microsoft Edge sends your typed text and a service token to a Microsoft cloud service over a secure HTTPS connection. The service token doesn't contain any user-identifiable information. A Microsoft cloud service then processes the text to detect spelling and grammar errors in your text. All your typed text that's sent to Microsoft is deleted immediately after processing occurs. No data is stored for any period of time. 
+By default, Edge provides spelling and grammar checking using **Microsoft Editor**. When using **Microsoft Editor**, Microsoft Edge sends your typed text and a service token to a Microsoft cloud service over a secure HTTPS connection. The service token doesn't contain any user-identifiable information. A Microsoft cloud service then processes the text to detect spelling and grammar errors in your text. All your typed text that's sent to Microsoft is deleted immediately after processing occurs. No data is stored for any period of time.   
 
-If you select **Basic** under the **Use writing assistance** toggle in `edge://settings/languages`, the service will perform only local spellchecking on the device, and no data will be sent to the cloud for spellchecking services. 
+To use **Basic** spelling and grammar checking: 
```

Comment:
```suggestion
If you select **Basic** instead of **Microsoft Editor**, Microsoft Edge will perform only local spellchecking on the device, and no data will be sent to the cloud for spellchecking services.

To use **Basic** spelling and grammar checking: 
```
Introduce the concept of Basic above the list of steps, and avoid the word 'service' for Basic.